images/macos/software-report/SoftwareReport.Common.psm1

```diff
@@ -71,8 +71,11 @@ function Get-FortranVersion {
 }
 
 function Get-ClangLLVMVersion {
-    $clangLLVMVersion = Run-Command "$(brew --prefix llvm)/bin/clang --version" | Select-Object -First 1 | Take-Part -Part 2
-    "Clang/LLVM $clangLLVMVersion"
+    $locationsList = @('', '$(brew --prefix llvm)/bin/')
```

Comment:
Expand doesn't work in a single quotes.

Reply:
yes, just want to show that this installation $(brew --prefix llvm) is installed via brew